### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by Jamelele**
Improve consistency in examples

### CODE REVIEW

**Comment by iCrawl** [comment]
If you could elaborate how this makes it better?
Also switching from `reject` to `console.error` doesn't seem better to me in any way.

File: src/structures/Guild.js [hunk 1]
```diff
@@ -573,7 +573,9 @@ class Guild {
    * be resolved, the user ID will be the result.
    * @example
    * // ban a user
-   * guild.ban('123123123123');
+   * guild.ban(user)
```
Comment (Jamelele) [inline src/structures/Guild.js]: 'user' instead of '123123123'

Comment (iCrawl) [inline src/structures/Guild.js]: But you don't explain what "user" is at all.
At the same time you add a promise where you pass the user if it resolves and console log it, but guild.ban can be used without the user being on the server initially, so you would have no user to resolve.

File: src/structures/Guild.js [hunk 2]
```diff
@@ -573,7 +573,9 @@ class Guild {
...
+   *  .then(user => console.log(`Banned ${user} from ${guild.name}`))
```
Comment (Jamelele) [inline src/structures/Guild.js]: .then() statement is also in line with other examples

File: src/structures/Guild.js [hunk 3]
```diff
@@ -585,9 +587,9 @@ class Guild {
    * @returns {Promise<User>}
    * @example
    * // unban a user
-   * guild.unban('123123123123')
+   * guild.unban(user)
    *  .then(user => console.log(`Unbanned ${user.username} from ${guild.name}`))
-   *  .catch(reject);
+   *  .catch(console.error);
```
Comment (Jamelele) [inline src/structures/Guild.js]: Use of 'console.error' is more in line with the rest of the examples

**Comment by Programmix** [comment]
Aside from what Crawl pointed out I feel like this PR is too specific. If you'd like to help us out with the documentation, that's great, but your PRs should cover a more diverse set of code. For example, I'm sure some of the consistency changes you made in this PR could've been applied in other areas too...

File: src/structures/Guild.js (hunk 1 above)
Comment (Jamelele) [inline src/structures/Guild.js]: I agree, expansion on what `user` is could be beneficial, but this is still an improvement on `123123123`.
As for bans being invoked before the user is in the guild, that's news to me. Do you know of a way to resolve a user outside of any guilds the bot is in?

**Comment by Gawdl3y** [comment]
The change from `reject` to `console.error` is :+1: for sure. The switch from an ID to a nondescript variable, however, isn't. You're right in that the current "ID" isn't very useful information, but neither is `user`. I'm thinking that the ID should be replaced with a placeholder string along the lines of `some user ID`, and a comment explaining that a user/member object are also usable.

File: src/structures/Guild.js (hunk 1 above)
Comment (1Computer1) [inline src/structures/Guild.js]: https://discord.js.org/#/docs/main/master/class/Client?scrollTo=fetchUser

**Pull request merged**